src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1397,4 +1731,9 @@ GlobeControls.prototype.reset = function reset() {
     this.updateCameraTransformation();
 };
 
+GlobeControls.prototype.setSceneLoaded = function setSceneLoaded() {
```

Comment:
Why is this function in `GlobeControls` prototype (ie : do you really want to expose it outside of this module)?